src/components/UsersActions/UsersActionsRemoveFromOrg.tsx

```diff
@@ -28,23 +28,26 @@ function FormDefault({
   const isPending = actionStatus === 'pending';
   return (
     <>
-      <p id={modalHeadingId(user)}>
-        Are you sure you want to remove <strong>{user.username}</strong> from
-        this organization?
+      <p id={modalHeadingId(user)} className="font-sans-md text-bold">
+        Are you sure you want to remove {user.username} from this organization?
```

Comment:
[non-blocking comment] Weird little edge case: there’s an “Unnamed user” in my user management table, which seems to quite literally not have a username:

<img width="464" alt="image" src="https://github.com/user-attachments/assets/384b6874-4b8a-459b-901d-ec6eb12185de">

Should we handle this in the UI? Like, “Are you sure you want to remove this account from this organization?”

Reply:
Good catch—we've been told that users with no names should not happen in a production environment. But we've been putting fallbacks where we find them just in case, since it happens in dev. I'll add that fallback text, thanks!